velero-plugins/main.go

```diff
@@ -14,7 +14,6 @@ func main() {
 		RegisterBackupItemAction("common-backup-plugin", newCommonBackupPlugin).
 		RegisterRestoreItemAction("common-restore-plugin", newCommonRestorePlugin).
 		RegisterBackupItemAction("is-backup-plugin", newImageStreamBackupPlugin).
-		RegisterRestoreItemAction("is-restore-plugin", newImageStreamRestorePlugin).
```

Comment:
I'm about to start working on a PR which updates imagestream.RestorePlugin to do real work, so it's probably best to remove this from the PR to avoid merge conflicts.